src/mapblock_mesh.cpp

```diff
@@ -1468,7 +1474,7 @@ void MeshCollector::append(const TileSpec &tile,
 	}
 
 	PreMeshBuffer *p = NULL;
-	for(u32 i=0; i<prebuffers.size(); i++)
+	for(u32 i = 0; i < prebuffers.size(); i++)
```

Comment:
If you are fixing spaces only here, you should change it to `for (u32 i = 0; i < prebuffers.size(); i++) {` instead (space between for and `(` and { on same line)